ophyd/areadetector/plugins.py

```diff
@@ -576,6 +576,7 @@ class ROIPlugin(PluginBase):
     _default_configuration_attrs = (PluginBase._default_configuration_attrs + (
         'roi_enable', 'name_', 'bin_', 'data_type_out', 'enable_scale')
     )
+    _default_read_attrs = ['enable', 'min_x', 'size_x', 'min_y', 'size_y', 'min_z', 'size_z']
```

Comment:
These have to match the attribute names of components, so `max_x` should be `max_xy.x`. Separately (in `__init__`), you can tell `max_xy.x` to call itself `max_x` like so:

```py
def __init__(self, *args, **kwargs):
    self.max_xy.x.name = '{name}_max_x'.format(name=name)
    ...
    super().__init__(*args, **kwargs)
```

The change will effect what the Component thinks its name is -- what shows up in `read()` and goes into the documents and thus all the downstream display/database etc. -- but it does not change its _attribute name_ so the reference `_default_read_attrs` and `read_attrs` etc. should stay the same.